### PR TITLE
Add configuration properties for all Url

### PR DESCRIPTION
**Pull request by Donatien26**

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.11</version>
+            <type>jar</type>
+        </dependency>
+        <dependency>
+            <groupId>commons-lang</groupId>
+            <artifactId>commons-lang</artifactId>
```
Comment (micedre) [inline core/pom.xml:162]: Do we need the 2 commons-lang ?

File: core/pom.xml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+        <dependency>
+            <groupId>org.keycloak</groupId>
+            <artifactId>keycloak-core</artifactId>
+            <version>${keycloak.version}</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.keycloak</groupId>
+            <artifactId>keycloak-services</artifactId>
```
Comment (micedre) [inline core/pom.xml:134]: Why do we need this ?

File: ear/pom.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <dependencies>
+        <dependency>
+            <groupId>fr.insee.keycloak</groupId>
+            <artifactId>keycloak-http-storage-provider-core</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
```
Comment (micedre) [inline ear/pom.xml]: manage this version by dependencymanagement in the parent module

**Review by micedre — CHANGES_REQUESTED** [review]

File: core/pom.xml (hunk 2 above)
Comment (Donatien26) [reply core/pom.xml:134]: Thanks to this dependencie we can create a HttpClient. That's why i include it :)

File: core/pom.xml (hunk 1 above)
Comment (Donatien26) [reply core/pom.xml:162]: nope i will remove commons lang 3 which is not used

**Review by micedre — APPROVED** [review]
Can we merge this ?

**Comment by Donatien26** [comment]
yep you can merge

**Pull request merged**